notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>To avoid confusion among readers, it may be helpful to include a figure that indicates where to select Ha, NII lines, and the slider when selecting "A better match." Furthermore, providing an example of how to achieve this would make the process clearer and more accessible to readers who may not be familiar with the terminology.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I am including a few screenshots.